gpu_direct_rdma_access.c

```diff
@@ -838,6 +866,66 @@ struct rdma_device *rdma_open_device_server(struct sockaddr *addr)
     return NULL;
 }
 
+int rdma_reset_server_device(struct rdma_task_attr *attr)
```

Comment:
it looks wrong to load the task_attr on the application logic
we started with a generic "rdma_reset_device(struct rdma_device *device)"
need to find some other way to pass this marker tasks to the post send logic..